Gemfile

```diff
@@ -2,3 +2,23 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in activeadmin_settings_cached.gemspec
 gemspec
+
+group :development do
+  gem 'rubocop'
+end
+
+group :test do
+  gem 'rails', '~> 4.2.5'
+  gem 'turbolinks'
+  gem 'rspec-rails'
+  gem 'rails-settings-cached', github: 'merqlove/rails-settings-cached'
+  gem 'activeadmin', github: 'activeadmin/activeadmin', branch: 'ab1d8d1a4a1af748ae7009d9d725193d780d0059'
+  gem 'coveralls', require: false
+  gem 'sass-rails'
+  gem 'sqlite3'
+  gem 'launchy'
+  gem 'database_cleaner'
+  gem 'capybara'
+  gem 'selenium-webdriver'
+  gem 'poltergeist'
```

Comment:
Yes. if you need dependency from github you should specify that in Gemfile. But not all deps should be in Gemfile
For example

``` ruby
# gemspec
spec.add_development_dependency "capybara"
spec.add_development_dependency "github_gem"

# Gemfile
group :test do
  gem 'github_gem', github: '...'
end